src/index.ts

```diff
@@ -98,6 +98,12 @@ export const isString = isOfBasicType('string')
  */
 export const isBoolean = isOfBasicType('boolean')
 
+/**
+ * Create a validator that asserts the passed argument is of type `'boolean'`,
+ * just like `typeof input == 'boolean'`.
```

Comment:
Oops... Should have been `symbol` instead of `boolean`.

Reply:
Nice catch. It's also missing in the README, where there's a list of `isNumber`, etc.